easybuild/easyconfigs/x/xtb/xtb-6.3.0-foss-2019a-pre.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'MesonNinja'
+
+name = 'xtb'
+version = '6.3.0'
```

Comment:
The current version is 6.2.2, the next will be 6.2.3, either way this version is wrong.